src/apple/config/mod.rs

```diff
@@ -174,6 +182,8 @@ pub struct Config {
     app: App,
     development_team: String,
     project_dir: String,
+    bundle_version: VersionTriple,
```

Comment:
`CFBundleVersion` can have `[3, inf)` components, right? I don't want to block things on implementing that, so could you slap a todo on this:?

Reply:
Wouldn't the build number typically be the 4th component? Letting people specify this separately doesn't amount to much without it.

Reply:
I was under the impression that it just got appended like `[major].[minor].[patch][build]` as opposed to `[major].[minor].[patch].[build]` but I could be completely wrong.

Reply:
Anyways, I can throw a todo in there and tackle it later